movies.json

```diff
@@ -1,16 +1,365 @@
+            "description": "A grumpy Grinch plots to ruin Christmas for the village of Whoville.",
+            "kid": true,
+            "rating": 4,
+            "new": false,
+            "poster": "https://m.media-amazon.com/images/M/MV5BMGY2YTVhNTItNGIwZi00YjA5LWI0M2YtMDI2YWUwNmY4OTE0XkEyXkFqcGdeQXVyMTYzMDM0NTU@._V1_.jpg"
+          },
+          {
+            "title": "Home",
+            "director": "Tim Johnson",
+            "release_year": 2015,
+            "genre": ["Animerad", "Äventyr", "Komedi"],
+            "actors": ["Jim Parsons", "Rihanna"],
+            "description": "An alien on the run from his own people makes friends with a girl. He tries to help her on her quest, but can be an interference.",
+            "kid": true,
+            "rating": 2,
+            "new": false,
+            "poster": "https://m.media-amazon.com/images/M/MV5BMjExOTQ4MDMyMV5BMl5BanBnXkFtZTgwMTE3NDM2MzE@._V1_.jpg"
+          },
+          {
+            "title": "The Boss Baby",
+            "director": "Tom McGrath",
+            "release_year": 2017,
+            "genre": ["Animerad", "Äventyr", "komedi"],
+            "actors": ["Alec Baldwin", "Steve Buscemi"],
+            "description": "",
+            "kid": true,
+            "rating": 2,
+            "new": false,
+            "poster": "https://m.media-amazon.com/images/M/MV5BNTVhMzdiMDgtODk3Zi00MGQyLTkxNGItOWJkZGQ3ZGZlYmE2XkEyXkFqcGdeQXVyODIyOTEyMzY@._V1_.jpg"
+          },
+          {
+            "title": "The Good Dinosaur",
+            "director": "Peter Sohn",
+            "release_year": 2015,
+            "genre": ["Äventyr", "Animerad", "Komedi"],
+            "actors": ["Jeffrey Wright", "Maleah Nipay-Padilla"],
+            "description": "In a world where dinosaurs and humans live side-by-side, an Apatosaurus named Arlo makes an unlikely human friend.",
+            "kid": true,
+            "rating": 5,
+            "new": false,
+            "poster": "https://m.media-amazon.com/images/M/MV5BMTc5MTg2NjQ4MV5BMl5BanBnXkFtZTgwNzcxOTY5NjE@._V1_FMjpg_UX1000_.jpg"
+          },
+          {
+            "title": "Diary of a Wimpy Kid",
+            "director": "Thor Freudenthal",
+            "release_year": 2010,
+            "genre": ["Komedi", "Drama"],
+            "actors": ["Zachary Gordon", "Robert Capron"],
+            "description": "Greg Heffley is a 12 year old who is fresh out of elementary and transitions to middle school, where he has to learn the consequences and responsibility to survive the year.",
+            "kid": true,
+            "rating": 3,
+            "new": false,
+            "poster": "https://m.media-amazon.com/images/M/MV5BMTg3NzQ2NDgyNF5BMl5BanBnXkFtZTcwMDc1NzIyMw@@._V1_FMjpg_UX1000_.jpg"
+          },
+          {
+            "title": "The Polar Express",
+            "director": "Robert Zemeckis",
+            "release_year": 2004,
+            "genre": ["Animerad", "Äventyr"],
+            "actors": [""],
```

Comment:
Please add actors if necessary.

Reply:
Fixed! :)

Reply:
Fixed the missing info. There are 10 upcoming movies but 5 that has their world premiere after new year